app/assets/javascripts/judge.js

```diff
@@ -147,6 +152,9 @@
           'value'    : el.value,
           'kind'     : kind
         };
+        if (kind == 'uniqueness') {
```

Comment:
Only a tiny thing but I'd like to maintain the style of using `===` in JS when testing for equality, and using `==` only when you need/want the ambiguity. It doesn't affect things at all in this case but it's a good habit to develop.